packages/gradient/src/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { LinearGradient } from 'expo-linear-gradient'
+import { styled, useDripsyTheme } from '@dripsy/core'
+
+type Props = Omit<React.ComponentProps<typeof LinearGradient>, 'colors'> & {
+  gradient?: string
+  colors?: string[]
+}
+
+const Grad = styled(
```

Comment:
One thing I noticed with the PR: the theme has a linear gradients prop, but there's no consumption of them here. I assume this component should destructure the variant prop and use this to check the theme for a given variant.

Reply:
Disregard this, I now see it with the gradients prop.